starter_code/index.html

```diff
@@ -3,18 +3,13 @@
   <head>
     <meta charset="utf-8">
     <title>React App</title>
-    <!-- FONT AWESOME -->
-    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
-    <!-- BOOTSWATCH -->
-    <link rel="stylesheet" href="http://bootswatch.com/journal/bootstrap.min.css">
-    <!-- JQUERY -->
-    <script src="https://code.jquery.com/jquery-3.0.0.min.js" integrity="sha256-JmvOoLtYsmqlsWxa7mDSLMwa6dZ9rrIdtrrVYRnDRH0=" crossorigin="anonymous"></script>
     <!-- REACT -->
     <script src="https://fb.me/react-15.2.0.min.js"></script>
     <script src="https://fb.me/react-dom-15.2.0.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.34/browser.min.js"></script>
     <!-- COMPONENTS -->
     <script type="text/babel" src="components.js"></script>
```

Comment:
You may need to put this JS at the bottom of the page so that `#app-container` is available when it loads.